packages/manager/modules/pci/src/projects/project/workflow/add/schedule/schedule.html

```diff
@@ -0,0 +1,193 @@
+<div class="container-fluid px-0">
+    <div class="row">
+        <oui-select-picker
+            class="d-inline-block col-md-6 col-xl-4 my-3"
+            data-ng-repeat="schedule in $ctrl.WORKFLOW_SCHEDULES"
+            data-name="{{ schedule.name }}"
+            data-description="{{ ::'pci_workflow_create_schedule_' + schedule.name + '_description' | translate }}"
+            data-model="$ctrl.schedule"
+            data-label="{{ ::'pci_workflow_create_schedule_' + schedule.name + '_title' | translate }}"
+            data-values="[schedule]"
+            data-variant="light"
+            data-required>
+        </oui-select-picker>
+    </div>
+    <div class="row" data-ng-if="$ctrl.schedule && $ctrl.schedule.name === 'custom'">
+        <div class="col-md-12">
```

Comment:
No need for this then 
```suggestion
        <div>
```

Reply:
Following @marie-j 's comment, you should remove the row too :)